Services/CustomPrefixResolver.cs

```diff
@@ -0,0 +1,33 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.Processors.TextCommands.Parsing;
+
+public class CustomPrefixResolver : IPrefixResolver
+{
+    private readonly IDatabaseService _database;
+
+    public CustomPrefixResolver(IDatabaseService database)
+    {
+        _database = database;
```

Comment:
I would say use using statements instead of injection, see the other comment I made for more info

Reply:
i dont understand